modules/Microsoft.Compute/virtualMachineScaleSets/.test/linux.min/dependencies.bicep

```
@@ -0,0 +1,90 @@
+@description('Optional. The location to deploy to')
```

```suggestion
@description('Optional. The location to deploy to.')
```

Probably in other modules as well